### PR TITLE
Add methods and data 

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Add methods
- [x] Add data to database

## Description of the task to be completed ↓↓ 

- Create blog app

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class User < ApplicationRecord
   has_many :posts, foreign_key: 'author_id'
+  has_many :comments
```
Comment (OyePriscilla) [inline app/models/user.rb:3]: - [ ] Please kindly add the `likes` association to this model in accordance to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)

File: app/models/post.rb [hunk 2]
```diff
@@ -1,3 +1,12 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
```
Comment (OyePriscilla) [inline app/models/post.rb:3]: - [ ] Please kindly add the likes association to this model in accordance to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @SamTush ✔️ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Set up associations between models ✔️
Implemented methods for all the models as expected ✔️

## Required Changes ♻️

- [ ] Please kindly fix the broken link in the tech stack section of your README file, I guess there is a typo error; `rubyonrails.org` not `rubyrails.org`. ℹ️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
Thank you @OyePriscilla for reviewing my PR😎

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @SamTush  

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️
- Associations well setup✔️
- methods well added ✔️ 
- No linter errors✔️

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 ✔️✔️

t

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SamTush** [comment]
thank you @tufoinnkuo10 for reviewing my PR😎

**Pull request merged**